cub/device/dispatch/dispatch_radix_sort.cuh

```diff
@@ -70,11 +70,12 @@ CUB_NAMESPACE_BEGIN
  * Upsweep digit-counting kernel entry point (multi-block).  Computes privatized digit histograms, one per block.
  */
 template <
-    typename                ChainedPolicyT,                 ///< Chained tuning policy
-    bool                    ALT_DIGIT_BITS,                 ///< Whether or not to use the alternate (lower-bits) policy
-    bool                    IS_DESCENDING,                  ///< Whether or not the sorted-order is high-to-low
-    typename                KeyT,                           ///< Key type
-    typename                OffsetT>                        ///< Signed integer type for global offsets
+    typename ChainedPolicyT,                 ///< Chained tuning policy
+    bool     ALT_DIGIT_BITS,                 ///< Whether or not to use the alternate (lower-bits) policy
+    bool     IS_DESCENDING,                  ///< Whether or not the sorted-order is high-to-low
+    typename KeyT,                           ///< Key type
+    typename OffsetT,                        ///< Signed integer type for global offsets
+    typename DecomposerT = detail::fundamental_decomposer_t>                        
```

Comment:
Add documentation stating that `DecomposerT` is considered as part of implementation detail and can be changed without bumping the major version.